docs/workflows/templates.md

```diff
@@ -0,0 +1,61 @@
+## Content Templates
+
+## Background
+
+Content Templates allow users to set a baseline of repository snapshots that help them control patching in predictable ways. In the future,
+users will be able to build images with a Content Template.
+
+### What is a Content Template?
+
+Content Templates are entities that can include snapshots from different Red Hat and custom repositories. 
+
+The content included in Content Templates is served at different distribution paths in pulp for Red Hat and custom since they are in different domains.
+
+For Red Hat content, the distribution path would be `/pulp/content/<red_hat_domain>/templates/<template_uuid>/`. For example, for this [Red Hat repository](https://cdn.redhat.com/content/dist/layered/rhel8/x86_64/ansible/2/os/), 
+the content would live at `/pulp/content/<red_hat_domain>/templates/<template_uuid>/content/dist/layered/rhel8/x86_64/ansible/2/os/`.
+
+For custom content, the distribution path would be `/pulp/content/<custom_domain>/templates/<template_uuid>/<repository_uuid>/`.
+
+### What happens on our end when a Content Template is created/updated/deleted?
```

Comment:
```suggestion
### What happens when a Content Template is created, updated, and deleted
```
"our end" is vague
Please don't use slash in text.
LEave off question marks because a title does not need to be a question, it can just be statement.